tests/unit/test_isort.py

```diff
@@ -2358,10 +2358,10 @@ def test_alphabetic_sorting_no_newlines() -> None:
 def test_sort_within_section() -> None:
     """Test to ensure its possible to force isort to sort within sections"""
     test_input = (
-        "from Foob import ar\n"
         "import foo\n"
         "from foo import bar\n"
         "from foo.bar import Quux, baz\n"
+        "from Foob import ar\n"
```

Comment:
I'm a little hesitant to make a change that would change the default styling with a single option set, as it would break isort's formatting guarantee: https://pycqa.github.io/isort/docs/major_releases/release_policy/#formatting-guarantees. Is there any chance you could restructure the change to not affect the existing test cases, possibly being behind it's own flag - and then the project can revisit in isort 6.0.0 next year?

Reply:
Yeah, I realized this was a bit problematic... Not sure how to do it without a flag, so I'll add a flag.